ospfd/ospf_interface.h

```diff
@@ -60,11 +60,6 @@ struct ospf_if_params {
 			 output_cost_cmd); /* Command Interface Output Cost */
 	DECLARE_IF_PARAM(uint32_t,
 			 retransmit_interval); /* Retransmission Interval */
-	DECLARE_IF_PARAM(uint8_t, passive_interface); /* OSPF Interface is
-							passive: no sending or
-							receiving (no need to
-							join multicast groups)
-							*/
 	DECLARE_IF_PARAM(uint8_t, priority); /* OSPF Interface priority */
```

Comment:
I disagree with taking one interface configuration option and pulling it out of the `ospf_if_params` struct while everything else is still in `ospf_if_params`. If you think the `ospf_if_params` struct is a bad idea in general, we should talk about that and we can change it for everything.

But either way it's not related to this patch and I don't see why it should be mixed in here.